### DISCUSSION TITLE
WinUI package doesn't appear in the output folder of Azure DevOps pipeline

### DISCUSSION THREAD

**Discussion by Juriyx** [unoplatform/uno#17004] [Q&A]
I tested locally in Visual Studio a command with these flags:
`/p:TargetFramework=net8.0-windows10.0.22621 /p:Configuration=Debug /p:Platform=x64 /p:PublishSignedPackage=true /p:AppxPackageDir="C:/temp/output/"`
Files were created as expected in an output folder:
![Local build](https://github.com/unoplatform/uno/assets/10002290/4f0d7cb1-ed4f-42ba-808c-a8fa149d46d9)
The same command and another one that I tried based on [this manual](https://learn.microsoft.com/en-us/windows/apps/package-and-deploy/ci-for-winui3?pivots=winui3-packaged-csharp)
`/p:AppxBundle=Never /p:AppxPackageDir="Packages\" /p:UapAppxPackageBuildMode=SideloadOnly /p:PackageCertificateThumbprint=A06F700867812633D67FF0EDB7683F07711463EC /p:PackageCertificatePassword=$(WinAppKey) /p:GenerateAppxPackageOnBuild=true`
gave me two *.msix packages under this path:
`D:\A\1\S\Console\App\obj\x64\Debug\net8.0-windows10.0.22621\MsixContent\MSIX`
![зображення](https://github.com/unoplatform/uno/assets/10002290/f1291e64-d87f-4b17-be8e-2757d4abcf61)

I used VSBuild@1 task for build in my pipeline.

Why the behavior differences and what can I do to get the same result on CI?

**Comment by agneszitte** [comment]
cc @jeromelaban

**Reply by jeromelaban** [reply]
It's unclear why this would be the result, but it's very likely caused by a difference in parameters. You can troubleshoot the [build by using binlogs](https://platform.uno/docs/articles/uno-builds-troubleshooting.html) to see what the difference is.

**Reply by Juriyx** [reply]
@jeromelaban Hello. Maybe this happens because I don't install Windows SDK and doesn't run Uno.Check in the previous tasks?
Although [this documentation](https://github.com/actions/runner-images/blob/main/images/windows/Windows2022-Readme.md) says that SDK is present

**Reply by jeromelaban** [reply]
@Juriyx It's unlikely that it's caused by a missing SDK, the build would fail entirely. Have you checked the binlog? Its likely going to tell where the appx files go.

**Reply by Juriyx** [reply]
@jeromelaban What search request in the log do I have to type?

**Reply by jeromelaban** [reply]
@Juriyx searching for `.appx` would be a start, to determine if there's actually a package being generated.
